features/bevragen/persoon/verblijfplaats/locatie/protocollering-antwoord.feature

```diff
@@ -0,0 +1,39 @@
+#language: nl
+
+Functionaliteit: protocollering van het antwoord voor verblijfplaats locatie
+
+  Rule: In het antwoord geleverde velden worden geprotocolleerd als de elementnummers volgens Logisch ontwerp BRP
+    Dit is een 6-cijferige code, met zo nodig voorloopnul voor categorieën.
+    Alleen velden die echt geleverd zijn, dus een waarde hebben in de response, worden opgenomen als geleverde rubriek
+    Ook alle velden die worden meegeleverd voor het bepalen van het verblijfplaats type worden opgenomen.
+    
+    Abstract Scenario: Protocolleren van geleverde velden voor <fields veld> voor een persoon met locatiebeschrijving
+      Gegeven de persoon met burgerservicenummer '000000036' heeft de volgende gegevens
+      | naam            | waarde     |
+      | anummer (01.10) | 5875306514 |
+      En de persoon heeft de volgende 'verblijfplaats' gegevens
+      | naam                               | waarde   |
+      | gemeente van inschrijving (09.10)  | 0628     |
+      | functie adres (10.10)              | W        |
+      | datum aanvang adreshouding (10.30) | 20141003 |
+      | datum ingang geldigheid (85.10)    | 20141003 |
+      En de 'verblijfplaats' heeft de volgende 'adres' gegevens
+      | naam                        | waarde                            |
+      | gemeente_code               | 0530                              |
+      | locatiebeschrijving (12.10) | Woonboot tegenover de Grote Sloot |
+      Als personen wordt gezocht met de volgende parameters
+      | naam                | waarde                          |
+      | type                | RaadpleegMetBurgerservicenummer |
+      | burgerservicenummer | 000000036                       |
+      | fields              | <fields>                        |
+      Dan is protocollering opgeslagen met de volgende gegevens
+      | anummer    | request_geleverde_rubrieken |
+      | 5875306514 | <geleverde rubrieken>       |
```

Comment:
Ik mis hier de kolom voor 'request_zoek_rubrieken' en ik mis in dit feature een rule voor 'request_geleverde_rubrieken'.

Reply:
kolom toegevoegd